linkis-public-enhancements/linkis-datasource/linkis-metadata-query/service/mysql/pom.xml

```diff
@@ -55,6 +56,18 @@
       </exclusions>
     </dependency>
 
+    <dependency>
+      <groupId>com.clickhouse</groupId>
+      <artifactId>clickhouse-jdbc</artifactId>
+      <version>${clickhouse.version}</version>
+      <scope>provided</scope>
```

Comment:
https://mvnrepository.com/artifact/ru.yandex.clickhouse/clickhouse-jdbc/0.3.1   this dependency is license under ASF2.0 
should be added to release package?

Reply:
resolved